### DISCUSSION TITLE
TCOD meaning

### DISCUSSION THREAD

**Discussion by TarpsMcGarps** [libtcod/libtcod#96] [Q&A]
What does tcod stand for as an acronym? I've searched everywhere but I can't find anything on the actual meaning.

**Comment by HexDecimal** [comment]
It stands for "The Chronicles of Doryen", a game the library was originally created for.  The RogueBasin article for it is [here](http://www.roguebasin.com/index.php?title=Doryen).  Some pieces of that project are open source which can be found in the libtcod samples or as the non-library projects hosted [here](https://github.com/libtcod).

Right now I usually use plain 'tcod' to refer to the main library or any of the ports.
